src/main/scala/org/ergoplatform/settings/Constants.scala

```diff
@@ -89,4 +89,8 @@ object Constants {
     * Keys and hashes are 256-bits long, values are boxes, so value size is dynamic.
     */
   object StateTreeParameters extends AvlTreeParameters(keySize = HashLength, valueSize = None, labelSize = HashLength)
+
+  def timeToTakeSnapshot(height: Int): Boolean = {
+    height % MakeSnapshotEvery == MakeSnapshotEvery - 1
```

Comment:
Why not `height % MakeSnapshotEvery == 0`? Please add the comment.

Reply:
moved to other place and documented